File: src/main/java/com/sudosoo/takeiteasy/dto/category/CreateCategoryRequestDto.java
package com.sudosoo.takeiteasy.dto.category;

import jakarta.validation.constraints.NotNull;
import lombok.*;

@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class CreateCategoryRequestDto {
    @NotNull(message = "카테고리 이름을 입력해 주세요.")
    private String categoryName ;

}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/application/dto/category/CreateCategoryRequestDto.kt
package com.sudosoo.takeItEasy.application.dto.category

import jakarta.validation.Valid
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

class CreateCategoryRequestDto(
    @NotBlank(message = "카테고리 이름을 입력해 주세요.")
    @field:Size(max = 20, message = "카테고리 이름은 20자를 넘을 수 없습니다.")
    val title : String)
